db/schema.rb

```diff
@@ -87,7 +87,7 @@
     t.uuid "accountable_id"
     t.decimal "balance", precision: 19, scale: 4, default: "0.0"
     t.string "currency", default: "USD"
-    t.virtual "classification", type: :string, as: "\nCASE\n    WHEN ((accountable_type)::text = ANY ((ARRAY['Account::Loan'::character varying, 'Account::Credit'::character varying, 'Account::OtherLiability'::character varying])::text[])) THEN 'liability'::text\n    ELSE 'asset'::text\nEND", stored: true
+    t.virtual "classification", type: :string, as: "\nCASE\n    WHEN ((accountable_type)::text = ANY (ARRAY[('Account::Loan'::character varying)::text, ('Account::Credit'::character varying)::text, ('Account::OtherLiability'::character varying)::text])) THEN 'liability'::text\n    ELSE 'asset'::text\nEND", stored: true
```

Comment:
You had updated to v16.x right?  Maybe this isn't related to the PG version after all.

Reply:
My bad, I haven't noticed this has snuck in again. But yes, I had upgraded to 16.3 and still get this schema change when I run `bin/setup`

Reply:
Interesting, I'll have to look into this again and see if there's a way to avoid in the future.